[2021-07-08 13:21:26,150] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.query_users_data 2021-07-08T13:21:23.549533+00:00 [queued]>
[2021-07-08 13:21:26,172] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.query_users_data 2021-07-08T13:21:23.549533+00:00 [queued]>
[2021-07-08 13:21:26,173] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 13:21:26,173] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-08 13:21:26,174] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 13:21:26,182] {taskinstance.py:1107} INFO - Executing <Task(PostgresOperator): query_users_data> on 2021-07-08T13:21:23.549533+00:00
[2021-07-08 13:21:26,188] {standard_task_runner.py:52} INFO - Started process 18306 to run task
[2021-07-08 13:21:26,194] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'bluepi_dag', 'query_users_data', '2021-07-08T13:21:23.549533+00:00', '--job-id', '107', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bluepi_dag.py', '--cfg-path', '/tmp/tmp8ey9lvqw', '--error-file', '/tmp/tmpd3aasbno']
[2021-07-08 13:21:26,201] {standard_task_runner.py:77} INFO - Job 107: Subtask query_users_data
[2021-07-08 13:21:26,257] {logging_mixin.py:104} INFO - Running <TaskInstance: bluepi_dag.query_users_data 2021-07-08T13:21:23.549533+00:00 [running]> on host 3cf0296804bd
[2021-07-08 13:21:26,316] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=natchanon
AIRFLOW_CTX_DAG_ID=bluepi_dag
AIRFLOW_CTX_TASK_ID=query_users_data
AIRFLOW_CTX_EXECUTION_DATE=2021-07-08T13:21:23.549533+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-08T13:21:23.549533+00:00
[2021-07-08 13:21:26,318] {postgres.py:68} INFO - Executing: 
            SELECT * FROM users;
        
[2021-07-08 13:21:26,327] {base.py:78} INFO - Using connection to: id: my_postgres_conn. Host: 35.247.174.171, Port: 5432, Schema: postgres, Login: exam, Password: ***, extra: {}
[2021-07-08 13:21:27,487] {dbapi.py:204} INFO - Running statement: 
            SELECT * FROM users;
        , parameters: None
[2021-07-08 13:21:27,872] {dbapi.py:212} INFO - Rows affected: 2
[2021-07-08 13:21:28,079] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=bluepi_dag, task_id=query_users_data, execution_date=20210708T132123, start_date=20210708T132126, end_date=20210708T132128
[2021-07-08 13:21:28,121] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-08 13:21:28,138] {local_task_job.py:151} INFO - Task exited with return code 0
